pyproject.toml

```diff
@@ -0,0 +1,65 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [
+    "setuptools>=60",
+    "setuptools-scm>=8.0"
+    ]
+
+[project]
+name = "ibm-fms"
+dynamic = ["version"]
+description = "Foundation Model Stack is a collection of components for development, inference, training, and tuning of foundation models leveraging PyTorch native components."
+authors = [
+  {name = "Antoni Viros i Martin", email= "aviros@ibm.com"},
+  {name = "Davis Wertheimer", email= "Davis.Wertheimer@ibm.com"},
+  {name = "Joshua Rosenkranz", email = "jmrosenk@us.ibm.com"},
+  {name = "Raghu Kiran Ganti", email= "rganti@us.ibm.com"},
+]
+license = "Apache-2.0"
+readme = "README.md"
+requires-python = ">=3.11,<3.13"
+keywords = ['ibm-fms']
+classifiers=[
+    "Development Status :: 3 - Alpha",
```

Comment:
Alpha sounds good, it's the most conservative, and it allows the project to change its API easily.  @JRosenkranz is this ok?